### PR TITLE
Add Ukrainian description of Glosario

### PR DESCRIPTION
**Pull request by olexandr-konovalov**
Translation of the text for the Glosario front page.

Please don't merge until it will be reviewed within the Ukrainian translators community.

### CODE REVIEW

**Review by ViktoriiaGrivina — APPROVED** [review]
All is good. Approved.

**Comment by olexandr-konovalov** [comment]
@ViktoriiaGrivina thank you for your first review :)

File: _config.yml [hunk 1]
```diff
@@ -289,6 +289,18 @@ languages:
       salang morago go fitlhelela kitso eo. Mokwadi o kgona go golaganya
       tlhaloso ya mafoko a a botlhokwa mo lekwalong la gagwe le maranyane a
       kaedi, a aka ranolang mafoko ao ka diteme tse di farologaneng.
+  - key: uk
+    name: Ukrainian
```
Comment (olexandr-konovalov) [inline _config.yml:293]: @ViktoriiaGrivina was looking at https://glosario.carpentries.org/ - in the list of languages on the right we have native language names, so we need to replace Ukrainian -> "Українська мова" або "Українська". Any preferences?

Comment (olexandr-konovalov) [reply _config.yml]: Had a look at some examples of glossaries online - one word "Українська" would be consistent.

**Comment by olexandr-konovalov** [comment]
@ViktoriiaGrivina made some more changes - just in case, could you please have another look? Thanks.

File: _config.yml [hunk 2]
```diff
@@ -289,6 +289,18 @@ languages:
...
+    title: Глосарій
+    blurb: >
+      "Глосарій" - це словник термінів, що використовуються в науці про дані.
+      Він доступний не тільки у вигляді вебсайту, але ще й у формі бібліотек з
+      відкритим кодом для [R](https://github.com/carpentries/glosario-r/) та
+      [Python](https://github.com/carpentries/glosario-py/). Автори навчальних
+      матеріалів можуть додавати до них посилання на терміни з словника для
+      пояснення тематики уроку та необхідних попередніх знань. Автори також
+      можуть використовувати функції бібліотеки для вставки гіперпосилань
+      на терміни та їх означення на різних мовах.
```
Comment (Herobread) [inline _config.yml:303]: Не критично, але [визначення](https://slovnyk.ua/index.php?swrd=%D0%B2%D0%B8%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%8F) буде краще, ніж [означення](https://slovnyk.ua/index.php?swrd=%D0%BE%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%8F) у цьому контексті.

**Review by Herobread — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 2 above)
Comment (olexandr-konovalov) [reply _config.yml]: Цілком згоден, дякую!

**Review by Herobread — APPROVED** [review]

**Comment by olexandr-konovalov** [comment]
@elletjies could you please also merge this - I think we can't see an overview page otherwise.

@ViktoriiaGrivina in case of any further changes, we will incorporate them later.

**Review by elletjies — APPROVED** [review]

**Pull request merged**